### PR TITLE
add endpoint for repo-init workflow

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

Schedule thoth_repo_init using endpoint

### CODE REVIEW

File: thoth/user_api/api_v1.py [hunk 1]
```diff
@@ -1110,6 +1112,23 @@ def schedule_kebechet_webhook(body: Dict[str, Any]) -> Tuple[Dict[str, Any], int
     return _send_schedule_message(payload, kebechet_trigger_message, KebechetTriggerContent)
 
 
+def initialize_repo(project_url: str):
+    """Schedule thoth init repo workflow, which creates a PR on the passed repository."""
+    try:
+        url_obj = url_parse.urlparse(project_url)
+    except ValueError:
+        return {"error": "Error processing project_url"}, 400
+    if len(url_obj.path[1:].split("/")) != 2:
```
Comment (fridex) [inline thoth/user_api/api_v1.py:1121]: If the repo-init workflow supports solely GitHub, user-api could check the repo initialization is for a GitHub repo and eventually fail.

**Comment by codificat** [comment]
/assign @fridex 
following up previous review

**Review by fridex — APPROVED** [review]
looks good to me 👍🏻 💯

**Pull request merged**